roles/terrible/.travis.yml

```diff
@@ -26,4 +26,5 @@ script:
   - ansible-playbook tests/test.yml -i tests/inventory --syntax-check
```

Comment:
Why did you add a .travis.yml file?
With the next issue of mine, this is going to be overridden.
Is this job also redundant?

Reply:
This is automatically generated from ansible-galaxy when we generated the basic role